frontend/assets/stylesheets/theme/_tones.scss

```diff
@@ -178,6 +178,12 @@
 /* Neutral
    ========================================================================== */
 
+.tone-white {
+    background-color: $white;
+}
 .tone-tint {
     background-color: $c-neutral7;
 }
+.tone-featured-tint {
+    background-color: $c-neutral6;
```

Comment:
Should this not be the same grey as the tone above? We should change `.tone-tint` to be one or the other across the board.